support-workers/src/main/scala/com/gu/emailservices/DigitalPackEmailFields.scala

```diff
@@ -154,11 +155,22 @@ class DigitalPackEmailFields(
     sfContactId: SfContactId,
     emailAddress: String,
     deliveryDate: Option[LocalDate] = None,
+    userAttributes: Option[JsonObject] = None,
   ): EmailFields = {
     val attributePairs = JsonToAttributes.asFlattenedPairs(fields.asJsonObject).left.map(
       error => throw new RuntimeException(s"coding error: $error")
     ).merge
-    EmailFields(attributePairs, Left(sfContactId), emailAddress, dataExtensionName, deliveryDate)
+    EmailFields(attributePairs, Left(sfContactId), emailAddress, dataExtensionName, deliveryDate, userAttributes)
+  }
+
+  private def wrap2[UserAttributes: Encoder.AsObject](
```

Comment:
Do we really need the wrap2 method? Can't we just do `userAttributes.map(_.asJsonObject)` at the call site?

Reply:
good spot, although I feel like both the wrap methods shouldn't really exist, they were kind of one of the strings I didn't pull at when I was doing the refactors
I will see if I can not add the second one now.